spec/models/book_spec.rb

```diff
@@ -29,12 +29,42 @@
     end
   end
 
-  describe ".sortable_fields" do
-    let(:book){ create(:book) }
+  describe "#sortable_fields" do
     it "returns a list of all fields that can be used for sorting" do
-      expect(book.sortable_fields).to eq(
+      expect(Book.sortable_fields).to eq(
```

Comment:
Hmm...I'm not sure where this method is coming from, but if it is `Book.sortable_fields` then that's a class method, and it should be `describe ".sortable_fields"`